tests/test_custom.py

```diff
@@ -0,0 +1,132 @@
+
+import sys
+import os
+import datetime as dt
+import logging
+import asyncio
+import csv
+from pathlib import Path
+
+# append parent directory pth
+CURRENT_FOLDER = os.path.dirname(os.path.realpath(__file__))
+PARENT_FOLDER = os.path.dirname(CURRENT_FOLDER)
+sys.path.append(PARENT_FOLDER)
+
+from dbdata import db_managers
+from v3data.constants import PROTOCOL_UNISWAP_V3
+#from v3data.common import hypervisor
+from v3data.hypes import impermanent_data, fees_yield
+
+
+CHAIN = "mainnet" # polygon
+mongo_srv_url = "mongodb://10.10.0.39:49154"
+db_name = "gamma_v1"
+collections = {
+    "static": {"id":True},
+    "returns":{"id":True}
+}
+
+
+
+async def simulate_query():
+
+    # start time log
+    _startime = dt.datetime.utcnow()
+
+    block = timestamp = 0
+    result = dict()
+    for days in [1,7,30]:
+        # init 
+        result[days] = dict()
+
+        # add ilg to result
+        all_dta = impermanent_data.ImpermanentData(period_days=days,protocol=PROTOCOL_UNISWAP_V3,chain=CHAIN)
```

Comment:
I prefer we just write data instead of dta, it's more readable and only 1 character longer.